cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/polling/task/ClouderaManagerCollectDiagnosticsListenerTask.java

```diff
@@ -69,7 +69,7 @@ private void digForFailureCause(ApiCommand apiCommand, List<String> errorReasons
                 .forEach(cmd -> digForFailureCause(cmd, childErrors, commandsResourceApi));
 
         if (childErrors.isEmpty() && StringUtils.isNotEmpty(apiCommand.getResultMessage())) {
-            String reason = String.format("Command [%s], with id [%.0f] failed: %s", apiCommand.getName(), apiCommand.getId(), apiCommand.getResultMessage());
+            String reason = String.format("Command [%s], with id [%s] failed: %s", apiCommand.getName(), apiCommand.getId(), apiCommand.getResultMessage());
```

Comment:
Is `apiCommand.getId()` really a `String`?

Reply:
does it matter?